src/UI/Settings/Notifications/Edit/NotificationEditView.js

```diff
@@ -33,6 +34,41 @@ var view = Marionette.ItemView.extend({
             model    : this.model,
             property : 'tags'
         });
+
+        if ('PushBullet' === this.model.get('implementation')) {
+            this._ifPushbullet();
+        }
+    },
+
+    _ifPushbullet : function() {
+        var $deviceIds = this.ui.pushbullet.find("#x-pushbullet-deviceIds");
+        var $button = this.ui.pushbullet.find("button");
+        var $deviceId = this.$("input[validation-name=DeviceId]");
+        var $accessToken = this.$("input[validation-name=ApiKey]");
+
+        function getDevices() {
+            var accessToken = $accessToken.val();
+            $.ajax({
+                type: 'GET',
+                url: 'https://api.pushbullet.com/v2/devices',
+                beforeSend: function(xhr) { xhr.setRequestHeader("Authorization", "Bearer " + accessToken); }
+            }).success(function(r) {
+                var $options = r.devices
+                    .filter(function(v) { return v.pushable; })
+                    .concat([{iden: '', active: true, nickname: 'All devices' }]) // Add extra option
+                    .map(function(v) { return $("<option>").val(v.iden).text(v.nickname + (v.active ? "" : "(Inactive)")); });
+                $deviceIds.html($options);
```

Comment:
Not a fan of this block, it mashes together processing and DOM manipulation and is awkward to follow/read.